src/modules/company/index.ts

```diff
@@ -35,21 +35,24 @@ export class Company {
    * faker.company.name() // 'Zieme, Hauck and McClure'
    */
   name(format?: number): string {
-    const formats = [
-      '{{name.lastName}} {{company.companySuffix}}',
-      '{{name.lastName}} - {{name.lastName}}',
-      '{{name.lastName}}, {{name.lastName}} and {{name.lastName}}',
-    ];
-
     if (format != null) {
       deprecated({
         deprecated: 'faker.company.name(format)',
-        proposed: 'faker.company.name()',
+        proposed: 'faker.company.name() or faker.helpers.fake(format)',
         since: '7.4',
         until: '8.0',
       });
     }
 
+    // ToDo: This `staticFormats` pattern should be removed in the future. It is only used to maintain backwards compatibility.
```

Comment:
```suggestion
    // TODO @Minozzzi 2022-08-22: This `staticFormats` pattern should be removed in the future. It is only used to maintain backwards compatibility.
```

Or you should just remove the whole todo comment and just create the issue after the merge 🤔

Reply:
Not all locales have patterns.